crates/matrix-sdk/src/docs/sync_running.md

```diff
@@ -1,20 +1,11 @@
 ## Synced State
-This method waits for the event spawned by a successful API call to be synced with the SDK's state. It therefore requires a running sync to complete. This functionality is implemented for:
-+ [Client::create_room][client_create_room]
-+ [Client::join_room_by_id][client_join]
-+ [Client::join_room_by_id_or_alias][client_join_alias]
-+ [Invited::accept_invitation][invited_accept]
-+ [Left::join][left_join]
-<!--
-    These are commented out because they link to private methods.
-    + [Common::join][common_join]
-    + [Common::leave][common_leave],
--->
+This method waits for the event spawned by a successful API call to be synced with the SDK's state. It therefore requires concurrent the receival of events. One can sync events using these methods:
```

Comment:
```suggestion
This method waits for the event spawned by a successful API call to be synced with the SDK's state. It therefore requires the concurrent receival of events. One can sync events using any these methods:
```

Reply:
if you edit the suggested change to `any of` I can apply it from the UI :D

Reply:
Haha, right. Done.